README.md

```diff
@@ -54,6 +54,96 @@ Model::search('query')
 ```
 
 
+## `with`
```

Comment:
With regards to the using results in the view, yes they are being used in the view as they need to be rendered in to the page and not client side generated as they are core parts of the  page content for SEO purposes. It also keeps our number of requests down to improve overall page performance.